crates/js-sys/src/lib.rs

```diff
@@ -2663,6 +2663,31 @@ extern "C" {
     pub fn validate(bufferSource: &JsValue) -> Result<bool, JsValue>;
 }
 
+// JSON
+#[wasm_bindgen]
+extern "C" {
+
+    #[derive(Clone, Debug)]
+    pub type JSON;
+
+    /// The `JSON.parse()` method parses a JSON string, constructing the
+    /// JavaScript value or object described by the string.
+    ///
+    /// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse
+    #[wasm_bindgen(catch, static_method_of = JSON)]
+    pub fn parse(text: &str) -> Result<JsValue, JsValue>;
+
+    /// The JSON.stringify() method converts a JavaScript value to a JSON string,
+    /// optionally replacing values if a replacer function is specified or
+    /// optionally including only the specified properties if a replacer array is
+    /// specified.
+    ///
+    /// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify
+    #[wasm_bindgen(static_method_of = JSON)]
+    pub fn stringify(obj: &JsValue) -> JsString;
```

Comment:
Unfortunately, `JSON.stringify` can throw errors, even without the replacer functions:

![json-lol](https://user-images.githubusercontent.com/74571/43969950-a6b559c0-9c80-11e8-869e-100bf6851a96.png)

Therefore, we should add the `catch` attribute and return `Result<JsString, JsValue>` here.